src/content/index.tsx

```diff
@@ -157,7 +157,7 @@ chrome.runtime.onMessage.addListener((message, sender, sendResponse) => {
 
       // Save the new state to local storage
       chrome.storage.local.set({ drawerOpen: !isOpen }, () => {
-        console.log('Drawer state saved:', !isOpen);
+        // console.log('Drawer state saved:', !isOpen);
```

Comment:
Can we remove these console logs instead of commenting them out?

Reply:
removed!